mtp_api/apps/core/management/commands/dump_for_ap.py

```diff
@@ -36,17 +35,18 @@ def handle(self, *args, **options):
 
         record_type = options['type']
         serialiser: Serialiser = Serialiser.serialisers[record_type]()
-        writer = csv.DictWriter(options['path'], serialiser.headers)
-        writer.writeheader()
-        records = serialiser.get_modified_records(after, before)
-        for record in records:
-            writer.writerow(serialiser.serialise(record))
+
+        with open(options['path'], 'wt') as jsonl_file:
+            records = serialiser.get_modified_records(after, before)
+
+            for record in records:
+                jsonl_file.write(json.dumps(serialiser.serialise(record), default=str, ensure_ascii=False))
```

Comment:
can you remind me what the `default=str` is for? is it the datetime type? to my mind, it's the serialisers' job to ensure that the output is in the correct format. will `default=str` here output the datetime in a timezone-aware machine readable format?

Reply:
That's right - a datetime value breaks the JSON output, and the default=str handles this.

There's likely a need for the analysis to parse the datetime from a string but I'd assume that this was a known in requesting JSON Lines format.

Reply:
so is the output timezone aware?

Reply:
ok i checked, and looks like it comes out _nearly_ in ISO format. i guess it's up to you what you prefer – while it's not the super standard format, it does have timezone info